src/main/java/com/target/ready/library/system/Controller/LibraryController.java

```diff
@@ -18,10 +18,20 @@ public class LibraryController {
     LibraryController(LibrarySystemService librarySystemService){
         this.librarySystemService=librarySystemService;
     }
-    @GetMapping("getBooks")
-    public List<Book> getAllBooks(){
-
-        return librarySystemService.getAllBooks();
+    @GetMapping("books_directory")
```

Comment:
Please auto format all the files using IntelliJ.

Reply:
sure will do that